services/worker/src/worker/main.py

```diff
@@ -140,10 +182,35 @@ def is_worker_alive(self, worker_loop_executor: OutputExecutor) -> bool:
         app_config = AppConfig.from_env()
         init_logging(log_level=app_config.common.log_level)
 
-        with QueueMongoResource(
-            database=app_config.queue.mongo_database, host=app_config.queue.mongo_url
-        ) as queue_resource:
+        init_logging(log_level=app_config.common.log_level)
+        # ^ set first to have logs as soon as possible
+        assets_directory = init_assets_dir(directory=app_config.assets.storage_directory)
```

Comment:
OK. It's weird to have exactly the same code in two files...

Reply:
not sure it's worth factoring this